roles/nginx/tasks/install.yml

```diff
@@ -25,3 +25,11 @@
     - { src: "roles/nginx/templates/server.conf", dest: "/etc/nginx/" }
     - { src: "roles/nginx/templates/nginx.conf", dest: "/etc/nginx/" }
     - { src: 'roles/nginx/templates/ports.conf', dest: '/etc/{{ apache_service }}/' , mode: '0644' }
+    - { src: 'roles/nginx/templates/uwsgi.service', dest: '/etc/systemd/system/' , mode: '0644' }
+
+- name: Let uwsgi running as {{ apache_user }} write log files
+  file: 
+      path: /var/log/uwsgi/app
+      state: directory
+      owner: "{{ apache_user }}"
+
```

Comment:
This should go into captiveportal/main.yml to ensure the log directory is created for captiveportal.log when runrole is used.

Reply:
uwsgi is a dependency for both captive portal, and admin Console. I think it belongs where it is.

Reply:
With the recent commit the logging moved to apache2 for captive portal and Admin Console doesn't use a log directory.